class.go

```diff
@@ -2,21 +2,57 @@ package netlink
 
 import (
 	"fmt"
+
+	"github.com/vishvananda/netlink/nl"
```

Comment:
I might be wrong, I think we usually do not import `netlink/nl` in the API go files.
For how annoying it may seem, you may need to redefine those `nl` pkg gnet stats here.